.idea/workspace.xml

```diff
@@ -1,7 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be committed, you probably miss `.idea` folder in `.gitignore` file.

Material to study: https://www.atlassian.com/git/tutorials/saving-changes/gitignore (plus `.gitignore` files in older projects)

Reply:
Done